.circleci/config.yml

```diff
@@ -49,18 +49,20 @@ jobs:
     steps:
       - install
       - run: yarn test
+      - run: yarn lint
       - run: yarn lint-css
       - run: yarn lint-ts
-      - run: yarn format-check
```

Comment:
This file was supposed to be deleted.

Reply:
I had it deleted before, but for some reason unlike dvc.org it still made a failed check on this site unlike dvc.org- maybe it needs some extra cloud config for some reason?